README.md

```diff
@@ -385,6 +355,13 @@ The official Meteor resources page can be found [here](https://www.meteor.com/to
 
 * [We Work Meteor](https://www.weworkmeteor.com/)
 
+## Related
+
+* [Awesome Meteor Developers](https://github.com/harryadelb/awesome-meteor-developers)
+* [Awesome Blaze](https://github.com/arggh/awesome-blaze)
+* [Awesome React Native Meteor](https://github.com/meteor-factory/awesome-react-native-meteor)
```

Comment:
it looks really outdated and there any a lot of new resources about those. are you sure it is the right resource to link to?

Reply:
You're right; I removed it. I may readd it if I ever find a better alternative.